regexes/device/mobiles.yml

```
@@ -4517,13 +4612,19 @@ Kazam:
 
 # Kempler & Strauss
 'Kempler & Strauss':
-  regex: 'Alumini3Plus|Alumini ?2'
+  regex: 'Alumini[_]?3[_]?Plus|Alumini ?[23]|KEMPLER_(TV|X)'
```

When only matching one character there is no need to use brackets:
```suggestion
  regex: 'Alumini_?3_?Plus|Alumini ?[23]|KEMPLER_(TV|X)'
```